### ISSUE TITLE
[BUG] RSS Doesn't seem to honor the full text setting

### ISSUE THREAD

**Issue by sheldonhull** [sunt-programator/CodeIT#108]
## Describe the bug

1. I tried to join discord, but the bug template link isn't working. Suggest regenerating a discord invite link for readme.
2. [rssFullText setting was configured](https://github.com/sheldonhull/sheldonhull.hugo/blob/8314cc5ca4334872ba9a3797b0203bec2e639138/config.yml#L223)
1. Home feed for entire site is empty. https://www.sheldonhull.com/index.xml
2. Section feed for blog only shows the summary section. https://www.sheldonhull.com/blog/index.xml

## Expected behavior

Full RSSText returned on feeds.

## My Attempts

I tried replacing with my prior feed layouts, but couldn't get them returning all contents.

## Expectation

1. Main feed for entire site setup to `https://sitename.com/index.xml` correctly shows all sections in rss feed.
2. Blog/Microblog sections show their section feeds.
3. Tag feeds show their tag feed. NOTE: THIS WORKS --> https://www.sheldonhull.com/tags/tech/index.xml

**Comment by victor-pogor** [comment]
Hi @sheldonhull,

Sorry for the late response to this issue. Does it still reproduce?

**Comment by sheldonhull** [comment]
@victor-pogor yes. I have resolved in my own rss layouts, along with webmentions support and ac:author tag fixes. If you'd like me to submit with my fixes as PR I'd be glad to. Pretty sure that default behavior in Hugo is to do this and requires override.

**Comment by victor-pogor** [comment]
@sheldonhull yeah, it would be great 😉
